multiversx_sdk_network_providers/proxy_network_provider.py

```diff
@@ -102,14 +102,14 @@ def get_tx() -> Dict[str, Any]:
             url = f"transaction/{tx_hash}?withResults=true"
             return self.do_get_generic(url).get('transaction', '')
 
-        process_status = None
+        status = None
```

Comment:
I think `status` can be renamed to `future_status` and `tx` to `future_tx`.

Reply:
Renamed